app.rb

```diff
@@ -0,0 +1,134 @@
+require './methods/create_person'
+require './methods/create_book'
+require './methods/create_rental'
+
+class App
+  def initialize
+    @students = []
+    @teachers = []
+    @books = []
+  end
+
+  def display_options
+    print "
+[1] List all books
+[2] List all people
+[3] Add a person
+[4] Add a book
+[5] Add a rental
+[6] List all rentals for a given person id
+[7] Exit\n
+Select an option by typing its number: "
+  end
+
+  def main
+    loop do
+      display_options
+      option = gets.chomp.to_i
+      if (1..7).include?(option)
+        run(option)
+      else
+        puts "Sorry, this option doesn't exist.\n"
+        sleep(0.8)
+      end
+      break if option == 7
+    end
+    puts 'Exiting...'
+  end
+
+  def run(option)
+    case option
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      add_person
+    when 4
+      add_book
+    when 5
+      add_rental
+    when 6
+      list_rental
+    end
+  end
+
+  def list_books
+    puts 'Booklist: '
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people
+    puts 'Teachers'
+    @teachers.each { |teacher| puts "Name: #{teacher.name}, Age: #{teacher.age}, ID: #{teacher.id}" }
+    puts 'Students'
+    @students.each { |student| puts "Name: #{student.name}, Age: #{student.age}, ID: #{student.id}" }
+  end
+
+  def add_person
+    print 'Do you want to add a student[1] or a teacher[2]? (Input the number): '
+    option = gets.chomp.to_i
+    case option
+    when 0
+      nil
```

Comment:
- [OPTIONAL] Great job creating your console app, is looking great! I see that you have an option '0' to go back, I would suggest moving the text `or [0] to return.` on line 69, this way the user would know how to go back before selecting any option.

Reply:
Thanks for the observation, I will implement that.